src/components/dropdown/dropdown-styles.scss

```diff
@@ -59,6 +80,12 @@
   }
 }
 
+.iconDropdown {
```

Comment:
I think this will change all the dropdowns on the page as its global 😬. Can we find a more specific way to target this?

Reply:
Being nested inside `.iconDropdown` class avoids this styles to affect other elements.

__Inside__ `iconDropdown` 👇 

![image](https://user-images.githubusercontent.com/6906348/48210931-aba38080-e378-11e8-8e1f-42788bc45f37.png)

__Not nested__ 👇 


![image](https://user-images.githubusercontent.com/6906348/48211102-0fc64480-e379-11e8-9103-573a8c65fd69.png)